test/drivers/test_cqrrpt.cc

```diff
@@ -22,41 +23,62 @@ class TestCQRRPT : public ::testing::Test
         int64_t col;
         int64_t rank; // has to be modifiable
         std::vector<T> A;
+        std::vector<T> A_row;
         std::vector<T> R;
         std::vector<int64_t> J;
         std::vector<T> A_cpy1;
         std::vector<T> A_cpy2;
         std::vector<T> I_ref;
 
-        CQRRPTTestData(int64_t m, int64_t n, int64_t k) :
-        A(m * n, 0.0), 
+        Layout layout;
+
+        CQRRPTTestData(int64_t m, int64_t n, int64_t k, Layout layout=Layout::ColMajor):
+        row(m),
+        col(n),
+        rank(k),
+        A(m * n, 0.0),
+        A_row(m * n, 0.0),
         R(n * n, 0.0),
-        J(n, 0),  
+        J(n, 0),
         A_cpy1(m * n, 0.0),
         A_cpy2(m * n, 0.0),
-        I_ref(k * k, 0.0) 
-        {
-            row = m;
-            col = n;
-            rank = k;
-        }
+        A_cpy3(m * n, 0.0),
```

Comment:
You're missing an `A_cpy3 `filed in the class -> results in a compilation error.

Reply:
Whoops, that should be removed